src/components/studies/StudyBuilder.tsx

```diff
@@ -77,12 +77,10 @@ const useStyles = makeStyles((theme: ThemeType) => ({
     },
   },
   introInfoContainer: {
-    display: 'flex',
-    flexDirection: 'row',
-    justifyContent: 'center',
-    alignItems: 'center',
+    textAlign: 'center',
     backgroundColor: '#FAFAFA',
-    height: '90vh',
+    height: '100vh',
+    paddingTop: '12vh',
```

Comment:
I don't think we need to have the vh units for top padding. Since the height of the element is not in vh. let's just stick to whatever margin is in the design in multiples of 8